.github/workflows/publish.yml

```diff
@@ -2,36 +2,40 @@ name: Publish
 
 on:
   push:
-    tags:
-      - "v*"
+    branches-ignore:
+      # No canary deploys for branches opened by dependabot
+      - 'dependabot/**'
 
 jobs:
   publish:
-    name: Publish to NPM
+    name: Make a release and publish to NPM
 
     runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
 
     steps:
       - uses: actions/checkout@v2
+
+      - name: Prepare repository
+        run: git fetch --unshallow --tags
+
       - uses: actions/setup-node@v2.4.0
         with:
           registry-url: "https://registry.npmjs.org"
           node-version: "15"
+
       - name: Install Node dependencies
         run: yarn --frozen-lockfile
+
       - name: Build
         run: yarn build
-      - name: Publish to NPM
-        run: npm publish
+
+      # npm config set scripts-prepend-node-path true
```

Comment:
What is this?

Reply:
I'll remove the comment. What this did was make sure that the version of node being used with `npm` and the version used when running `npm run SCRIPT_NAME` would be the same. I thought this might fix the issue (around needing 2 auth variables to be set), but it unfortunately didn't work.

https://github.com/npm/npm/issues/15398